source/configure/push-notification-server-configuration-settings.rst

```diff
@@ -0,0 +1,99 @@
+| - The TPNS is provided for testing push notifications prior to compiling your own service. Ensure you’re familiar with its `limitations          |
+|   <https://docs.mattermost.com/deploy/mobile-hpns.html#test-push-notifications-service-tpns>`__. Review the                                      |
+|   `mobile push notifications <https://docs.mattermost.com/deploy/mobile-hpns.html>`__                                                            |
+|   and `mobile apps <https://docs.mattermost.com/deploy/build-custom-mobile-apps.html>`__ documentation, including guidance on compiling your own |
+|   mobile apps and MPNS, before deploying to production.                                                                                          |
+| - To confirm push notifications are working, connect to the `Mattermost iOS App <https://apps.apple.com/us/app/mattermost/id1257222717>`__       |
+|   available on the App Store, or the `Mattermost Android App <https://play.google.com/store/apps/details?id=com.mattermost.rn>`__ available on   |
+|   Google Play.                                                                                                                                   |          
++-----------------------------------------------------------------+--------------------------------------------------------------------------------+
+
+Maximum notifications per channel
+---------------------------------
+
+|all-plans| |self-hosted|
+
+*Available in legacy Enterprise Edition E10/E20*
+
++-----------------------------------------------------------------+--------------------------------------------------------------------------------------+
+| Maximum total number of users in a channel before @all, @here,  | - System Config path: **Environment > Push Notification Server**                     |
+| and @channel no longer send notifications to maximize           | - ``config.json setting``: ``".TeamSettings.MaxNotificationsPerChannel: 1000",``     |
+| performance.                                                    | - Environment variable: ``MM_EMAILSETTINGS_MAXNOTIFICATIONSPERCHANNEL``              |
+|                                                                 |                                                                                      |
+| Numerical input. Default is 1000.                               |                                                                                      |
++-----------------------------------------------------------------+--------------------------------------------------------------------------------------+
+| **Note**: We recommend increasing this value a little at a time, monitoring system health by tracking `performance monitoring metrics                  |
+| <https://docs.mattermost.com/scale/performance-monitoring.html>`__, and only increasing this value if large channels have restricted permissions       |
```

Comment:
Not sure if this link is rendering.

Reply:
Interesting. I've confirmed it's working locally and in the preview. I'll also double-check once these changes are merged into production.

Reply:
I suspect it's just weird GitHub things, so thank you for checking!